app/components/search.tsx

```diff
@@ -79,7 +79,8 @@ export default function Search({onSiteAnswersRef, openQuestionTitles, onSelect}:
     }
   }
 
-  const handleChange = debounce(searchFn, 500)
+  // const handleChange = debounce(searchFn, 300)
+  const handleChange = searchFn
```

Comment:
have you tested in Firefox and on mobile browsers? debounce was added because search was still running for out-of-date queries while typing additional letters and then it took up to 2 seconds of blinking results until the final results were shown

to search without delay, you can always press Enter

not sure why I changed it to 500ms at the same time then adding the Enter trigger,, probably left over by mistake while testing - feel free to use 300 or even 100ms, but I am not sure about the zero debounce

Reply:
Sorry, I had this PR as a draft because it's not ready for review yet. Probably should have just dropped a message in discord. I have a demo at https://stampy-ui.ccstan99.workers.dev/ so we can test it on Firefox and mobile. I didn't see the blinking results on Firefox when I tested it on my machine but don't expect it to have magically fixed for people who were seeing that either.

Reply:
OK, I set the debounce to 100ms for now.